Calculator/Calculator/Model/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by Baem on 2022/09/20.
+//
+
+struct CalculatorItemQueue {
+    private var queue: [CalculatorItem] = []
+    var isEmpty: Bool {
+        return queue.isEmpty ? true : false
+    }
```

Comment:
반환형이 `Bool`타입이면 조건 자체를 리턴하면 가독성이 더 좋아집니다.

```suggestion
    var isEmpty: Bool {
        return queue.isEmpty
    }
```

Reply:
넵 감사합니다 확인했습니다!